### ISSUE TITLE
Bug in eigen redstone patch

### ISSUE THREAD

**Issue by Pulverizer** [SpongePowered/Sponge#3353]
**I am currently running**

- SpongeVanilla version: latest stable-7 in a dev environment
- Java version: 8 build 251
- Operating System: Windows 10

- Plugins/Mods:

**Issue Description**
been debugging why eigen redstone doesn't fix everything that panda redstone does and found this:

![](https://i.imgur.com/ba9M7Wd.png)

^ with this not all dispensers fire

![](https://i.imgur.com/ZuteNMB.png)

^ without the comparators it fires all the dispensers

is this something that sponge will fix or will the patch creator need to provide an update?

TL;DR: connecting a comparator signal extender to a redstone circuit causes the eigen patch to bug.

**Comment by ImMorpheus** [comment]
Hey! You seem to be running an unsupported version (API 7) of SpongeAPI. Support for API 7 ended in January 2022.

API-7 is now end-of-life. We will not be issuing any more API updates for version 7 of our API[^1]

As part of our legacy cleanup we are closing issues relating to API7/1.12.2[^2]
___

Eigen redstone was removed from Sponge

[^1]: https://forums.spongepowered.org/t/status-update-8th-january-2022-spongeapi-8-released/39861#minecraft-1122-spongeapi-7-10
[^2]: https://forums.spongepowered.org/t/sponge-status-update-28th-may-2022/40135#api-71122-community-support-8
